#Problem Set 3 (Due on Nov 15th)

In [22]:
import numpy as np

##1.0
Briefly describe the location and faults of the Campo Felice area and how the basin formed in recent geologic history.

<img src="https://github.com/mich1803/EQML-ProblemSets/blob/main/media/CampoFeliceFieldTrip081024/2.jpeg?raw=true">

Campo Felice is a mountain basin in the Apennines of central Italy, in the Abruzzo region. It formed due to stretching and pulling apart of the Earth's crust along faults, mainly the Ovindoli-Pezza and Campo Felice faults. These faults created a “graben,” or sunken block of land, as the crust extended and broke. This process has been ongoing for thousands of years and is why the area is still prone to earthquakes.

<img src="https://github.com/mich1803/EQML-ProblemSets/blob/main/media/CampoFeliceFieldTrip081024/3.jpeg?raw=true" width=33%>
<img src="https://github.com/mich1803/EQML-ProblemSets/blob/main/media/CampoFeliceFieldTrip081024/8.jpeg?raw=true" width=33%>
<img src="https://github.com/mich1803/EQML-ProblemSets/blob/main/media/CampoFeliceFieldTrip081024/9.jpeg?raw=true" width=33%>


##2.0
Determine the orientation of the Campo Felice fault and estimate its recent slip rate using your field observations and published data. In addition to data you collect in the field, I can suggest that you refer to this paper, which you can find on our course elearning page (Schirripa Spagnolo, G., Mercuri, M., Billi, A., Carminati, E., & Galli, P., Tectonics, 2021). Give the fault strike and dip and state clearly the estimate you used for the slip rate (note that sometimes authors use the term ‘throw rate’ to refer to slip rate).

Based on the Schirripa Spagnolo et al. (2021) paper and on-field measuring:

1. **Fault Orientation**:
   - The MCF (Monte Cefalone) and MOF (Monte Orsello) faults strike NW-SE (azimuth = 208° = strike + 90°).
   - Both faults dip steeply toward the SW, with an approximate dip angle of 41°.

2. **Slip Rate (Throw Rate)**:
   - The estimated slip (or throw) rate for the Monte Cefalone fault (MCF) is around **0.8 to 1.3 mm/year** (let's take 1 ± 0.3 mm/year ), as measured over the past 18,000 years. This rate reflects the average vertical movement along the fault and aligns with throw rates typical for active normal faults in the central Apennines.

In [23]:
dip = 41 #degrees
alpha = 208 #degrees
srate = 0.001 #m/year

##3.0
Let us assume that motion of the Campo Felice fault can be described by motion of Monte Cefalone relative to the Campo Felice basin. Let us further assume that this motion can be described by rotation around an Euler pole, consistent with standard ideas in Plate Tectonic Theory, and that this motion is perpendicular to the strike of the Campo Felice fault. Follow the standard convention that the positive pole of rotation intersects Earth’s surface at the location that describes counter clockwise motion of Monte Cefalone relative to the Campo Felice basin. To obtain the rotation vector we need the linear velocity vector v for Monte Cefalone relative to the Campo Felice basin and the location vector.

###3.1 Give the position vector at Campo Felice in Spherical Polar Coordinates

**Position Vector in Spherical Polar Coordinates**

Let:
- $ R $ be the radius of the Earth, approximately 6371 km.
- Latitude $ \phi $ and longitude $ \lambda $ be the coordinates of Campo Felice.

Assuming Campo Felice is located near:
- Latitude $ \phi \approx 42.2167^\circ $ N
- Longitude $ \lambda \approx 13.45^\circ $ E

**Position Vector Components**

In spherical polar coordinates:
- $ r = R \approx 6371 \, \text{km} $
- $ \theta = 90^\circ - \phi = 90^\circ - 42.2167^\circ = 47.7833^\circ $ (colatitude)
- $ \lambda = 13.45^\circ $ (longitude)



In [24]:
R = 6371 * 1e3 #m
CF = (42.2167, 13.45) #Latitude (degrees), Longitude (degrees)
CFspc = (R, 90-CF[0], CF[1]) #r (m), theta (colatitude, degrees), lambda (longitude, degrees)
print("Position Vector at Campo Felice in Spherical Polar Coordinates: ({:.1f} km, {:.2f} degrees, {:.2f} degrees)".format(CFspc[0]/1e3, CFspc[1], CFspc[2]))

Position Vector at Campo Felice in Spherical Polar Coordinates: (6371.0 km, 47.78 degrees, 13.45 degrees)


Thus, the **position vector $ \vec{r} $ at Campo Felice in spherical polar coordinates** is:
$$
\vec{r} = (r, \theta, \lambda) = (6371 \, \text{km}, 47.7833^\circ, 13.45^\circ)
$$

###3.2 Give the position vector at Campo Felice in Cartesian Coordinates

To express the position vector at Campo Felice in Cartesian coordinates, we’ll convert from spherical polar coordinates to Cartesian $(x, y, z)$.

**Cartesian Coordinates Formulas**

The Cartesian coordinates $(x, y, z)$ are given by:


$$
x = r \cos \theta \cos \lambda
$$
$$
y = r \cos \theta \sin \lambda
$$
$$
z = r \sin \theta
$$

In [25]:
CFcc = (CFspc[0]*np.cos(np.deg2rad(CFspc[1]))*np.cos(np.deg2rad(CFspc[2])),
        CFspc[0]*np.cos(np.deg2rad(CFspc[1]))*np.sin(np.deg2rad(CFspc[2])),
        CFspc[0]*np.sin(np.deg2rad(CFspc[1])))
print("Position Vector at Campo Felice in Cartesian Coordinates: ({:.1f} km, {:.1f} km, {:.1f} km)".format(CFcc[0]/1e3, CFcc[1]/1e3, CFcc[2]/1e3))

Position Vector at Campo Felice in Cartesian Coordinates: (4163.5 km, 995.7 km, 4718.4 km)


**Position Vector in Cartesian Coordinates**

Thus, the **position vector $\vec{r}$ at Campo Felice in Cartesian coordinates** is approximately:
$$
\vec{r} = (x, y, z) \approx (4162.5 \, \text{km}, 995.7 \, \text{km}, 4718.4 \, \text{km})
$$

###3.3 Give the velocity vector at Campo Felice in Cartesian Coordinates

In [26]:
Vx = srate * np.cos(np.deg2rad(alpha)) * np.sin(np.deg2rad(CF[0]))
Vy = srate * np.sin(np.deg2rad(alpha)) * np.sin(np.deg2rad(CF[0]))
Vz = srate * np.cos(np.deg2rad(CF[0]))
V = (Vx, Vy, Vz)

print("Velocity Vector at Campo Felice in Cartesian Coordinates: ({:.5f} mm/yr, {:.5f} mm/yr, {:.5f} mm/yr)".format(V[0]*1e3, V[1]*1e3, V[2]*1e3))
print("Magnitude of Velocity Vector: {:.2f} mm/yr".format(np.linalg.norm(V)*1e3))


Velocity Vector at Campo Felice in Cartesian Coordinates: (-0.59328 mm/yr, -0.31546 mm/yr, 0.74061 mm/yr)
Magnitude of Velocity Vector: 1.00 mm/yr


##4.0
Now let’s estimate the rotation vector that can describe motion of the Campo Felice fault. As you know, the plate tectonic velocity vector v can be determined from the rotation vector $\omega$ and position vector $r$ as $v$ = $\omega$ x $r$. Often, we know $\omega$ and $r$. In this case, we know
$v$ and $r$ from our field work and literature research. Keep in mind that vector cross products require consistent units and that the magnitude of the resulting vector is given by the product of the magnitudes of the vectors in the cross product. Determine the rotation vector $\omega$.

###4.2 Give the rotation vector ω in Cartesian Coordinates

We can compute ω doing:

$$
\omega = \frac{\vec{R} \times \vec{V}}{|R|^2}
$$

In [27]:
omega_cc = np.cross(CFcc, V) # m^2 / yrs
omega_cc /= np.linalg.norm(CFcc)**2
print("Rotation Vector in Cartesian Coordinates: ({:.2} rad/yr, {:.2} rad/yr, {:.2} rad/yr)".format(omega_cc[0], omega_cc[1], omega_cc[2]))
print("Magnitude of Rotation Vector: {:.2} rad/yr".format(np.linalg.norm(omega_cc)))

Rotation Vector in Cartesian Coordinates: (5.5e-11 rad/yr, -1.4e-10 rad/yr, -1.8e-11 rad/yr)
Magnitude of Rotation Vector: 1.6e-10 rad/yr


###4.1 Give the rotation vector ω in Spherical Polar Coordinates

now we can convert it in SPC:

$$
|\omega|, \quad  \omega_\theta = \text{arccos}(\frac{\omega_z}{|\omega|}), \quad \omega_\phi = \text{arctan}(\frac{\omega_y}{\omega_x})
$$

In [28]:
omega_spc = (np.linalg.norm(omega_cc), np.rad2deg(np.arccos(omega_cc[2]/np.linalg.norm(omega_cc))), np.rad2deg(np.arctan2(omega_cc[1], omega_cc[0])))
print("Rotation Vector in Spherical Polar Coordinates: ({:.2} rad/yr, {:.2f} degrees, {:.2f} degrees)".format(omega_spc[0], omega_spc[1], omega_spc[2]))

Rotation Vector in Spherical Polar Coordinates: (1.6e-10 rad/yr, 96.55 degrees, -69.28 degrees)
